### DISCUSSION TITLE
Can't bootstrap Talos

### DISCUSSION THREAD

**Discussion by eneko-perez** [siderolabs/talos#7902] [Q&A]
Hi all,

I'm trying to bootstrap Talos following the instructions here for Openstack (https://www.talos.dev/v1.5/talos-guides/install/cloud-platforms/openstack/). However, after adding one of the control plane ip addresses to talosconfig as stated in the docs and then running the bootstrap command, I get the following error:

error executing bootstrap: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: tls: failed to verify certificate: x509: certificate signed by unknown authority"

I don't know how to troubleshoot this and it's driving me a bit crazy. Any ideas what can be wrong here?

Thank you in advance.

**Comment by smira** [comment]
make sure you're using correct `talosconfig` or context, `talosctl config info` to help

if you have `talosconfig` generated as a separate file (not in the default location), pass it as `--talosconfig=./talosconfig`

**Reply by eneko-perez** [reply]
Hi. Yes, I forgot to mention that, that's exactly what I do. I define the node and the endpoint as the control plane instance, one of them, and that's where I get the message from. I can't bootstrap that specific control plane node (any of them).

**Reply by revoltez** [reply]
@eneko-perez how did you solve it?
